### DISCUSSION TITLE
Firebase crashlytics sdk 17.00 + is it supported be react-native-firebase/crashlytics

### DISCUSSION THREAD

**Discussion by ANIBIT14** [invertase/react-native-firebase#4140] [Q&A] [answered]
Firebase crashlytics has started giving warning :-
Please upgrade to the generally available Firebase Crashlytics SDK to continue getting crash reports in the Firebase console past November 15, 2020.

Disclosure in their docs - 
The Fabric SDK is now deprecated and will continue reporting your app's crashes until November 15, 2020. On this date, the Fabric SDK and old versions of the Firebase Crashlytics SDK will stop sending crashes for your app. To continue getting crash reports in the Firebase console, make sure you upgrade to the Firebase Crashlytics SDK versions 17.0.0+ for Android, 4.0.0+ for iOS, and 6.15.0+ for Unity.
https://firebase.google.com/docs/crashlytics/get-started?authuser=0&platform=android

So wanted to know will we betting an update for react-native-forebase/crashlytics so this will be rectified or is it already rectified

**Comment by Salakar** [comment, marked as answer]
Already rectified on the latest release 👍

**Reply by ANIBIT14** [reply]
Thanks for the information
